plutus-ledger/src/Ledger/Test.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE DataKinds           #-}
```

Comment:
Why not put this file in the `test` directory?

Reply:
Because it doesn't contain tests, it contains utilities useful for testing. Just like `Plutus.Contract.Test`. The `test` directories we have are not structured in a way that's useable from another package, they are all the same.